lib/config/screen.ex

```diff
@@ -13,6 +13,7 @@ defmodule ScreensConfig.Screen do
           | :elevator_v2
           | :gl_eink_v2
           | :pre_fare_v2
+          | :on_bus_v2
```

Comment:
nit: Keeping these alphabetized would be nice. (+L28, +L38, +L50)

Reply:
everything is alphabetized now